two_factor/gateways/twilio/gateway.py

```diff
@@ -52,7 +53,14 @@ def make_call(self, device, token):
                                  url=uri, method='GET', timeout=15)
 
     def send_sms(self, device, token):
-        body = gettext('Your authentication token is %s') % token
+        """
+        send sms using custom TWILIO_MESSAGE from settings.py
+        """
+        body = getattr(
+            settings,
+            'TWILIO_MESSAGE',
```

Comment:
I think a better name for this setting would be `TWO_FACTOR_SMS_MESSAGE`. It should also be documented so users know they can change it.